cli/cli.go

```diff
@@ -0,0 +1,36 @@
+package cli
```

Comment:
Like we talked about at lunch, if we followed the official go conventions this would live in `internal/cli` because this isn't designed to be used outside of this project.

Reply:
@jazzdan @nicks thoughts on this new directory structure? am i missing something?